edsdme/scripts/rewriteLinks.js

```diff
@@ -102,15 +114,16 @@ function setLocale(url) {
   if (!domainConfig) return;
   const localeFromMap = domainConfig.localeMap[currentPageLocale];
   if (!localeFromMap) return;
-  const pathParts = url.pathname.split('/').filter(Boolean);
+  let newPath;
```

Comment:
@SonjaPopovic this comment is actually for the lines above, sorry for missing that on the earlier PR: why do we need "localesToSkip"? If a locale is not configured in the domainMap, localeFromMap will be undefined, we do not need any additional logic.